jsonschema-generator-parent/pom.xml

```diff
@@ -134,10 +134,10 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <file.encoding>UTF-8</file.encoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
**suggestion**: Changing these settings and then not using them feels somewhat pointless.
Fair enough to switch to the `release`, but then let's please introduce a property for that again to be consistent.
```suggestion
        <maven.compiler.release>17</maven.compiler.release>
```